components/builder-db/src/config.rs

```diff
@@ -65,6 +73,28 @@ impl fmt::Display for DataStoreCfg {
             None => connect,
         };
         connect = format!("{}@{}:{}/{}", connect, self.host, self.port, self.database);
+        let mut opts = Vec::new();
+
+        if let Some(ref m) = self.ssl_mode {
+            opts.push(format!("sslmode={}", m));
```

Comment:
Do we also need to propagate this to IntoConnectParams below?

Reply:
From what I can tell, that isn't possible as you have to create special types that can read/write on ssl connections. I'm not sure why the 2 pools use different types.

Reply:
specifically, the problematic bits look like they are here:
https://github.com/habitat-sh/builder/blob/master/components/builder-db/src/pool.rs#L45-L47

the ssl-ness is in a different parameter

Reply:
Yeah there are 2 pools .. one for Diesel queries, and one for non-Diesel (legacy).
So we should probably fix that line above with the way we have the Diesel pool creation at https://github.com/habitat-sh/builder/blob/master/components/builder-db/src/diesel_pool.rs#L37